package.json

```diff
@@ -8,7 +8,7 @@
     "test": "jest",
     "test-e2e": "yarn build && node -r @babel/register ./config/scripts/CheckBuildsExist.js && cp -R ./build/* ./src/electron && cross-env NODE_ENV=test testcafe electron:./src/electron ./tests-e2e && rm -rf ./src/electron/dist ./src/electron/main.js",
     "tsc": "cross-env NODE_ENV=development ./node_modules/.bin/tsc",
-    "checks": "concurrently 'yarn lint' 'yarn test' 'yarn tsc'",
+    "checks": "concurrently \"yarn lint\" \"yarn test\" \"yarn tsc\"",
```

Comment:
should we include the e2e tests here? Or are they too much trouble to setup in CI for now?

Reply:
Yep that's a good call. Will add.